software/webapp/src/test/java/brooklyn/entity/dns/geoscaling/GeoscalingWebClientTest.java

```diff
@@ -42,8 +42,8 @@
     
     private final static String USERNAME = "cloudsoft";
     private final static String PASSWORD = "cl0uds0ft";
-    private final static String PRIMARY_DOMAIN = "domain"+((int)(Math.random()*10000))+".test.org";
-    private final static String SUBDOMAIN = "subdomain"+((int)(Math.random()*10000));
+    private final static String PRIMARY_DOMAIN = "domain"+((int)(Math.random()*10000) + 10000)+".test.org";
+    private final static String SUBDOMAIN = "subdomain"+((int)(Math.random()*10000) + 10000);
```

Comment:
Hmm, I don't think that's necessary: the existing (what, four?) `*.test.org` domains look like remnants of previous failed test runs and should be deleted.